src/GraphQL/Schemas/Traits/HasAuthorizableFields.php

```diff
@@ -2,12 +2,21 @@
 
 namespace Enjin\Platform\GraphQL\Schemas\Traits;
 
+use Enjin\Platform\Middlewares\OperationDefinitionNodeStore;
+
 trait HasAuthorizableFields
 {
     public function getFields(): array
     {
         $fields = parent::getFields();
 
-        return collect($fields)->filter(fn ($field) => !($field['authRequired'] ?? false))->all();
+        if (!config('enjin-platform.auth')) {
+            return $fields;
+        }
+
+        return collect($fields)
+            ->filter(fn ($field) => auth()->check() || !in_array(OperationDefinitionNodeStore::getOperationName(), $field['excludeFrom'] ?? []))
```

Comment:
Should we combine the two filters into one for better performance?

Reply:
I kept them separate for readability, and to make it clearer we have two types of filtering, globally with the `authRequired` field arg, and also the `excludeFrom` arg for only hiding on certain queries and mutations when not authenticated, but can be combined if you think it will still be clear enough 🙂

Reply:
I think you can just combine them, just arrange the code in better format so it's easy to read.